benchmarking/benchmark.py

```diff
@@ -225,15 +226,19 @@ def get_times_e3fp(X: pd.DataFrame):
                 mol = PropertyMol(mol)
                 mol.SetProp("_SMILES", smiles)
 
-                # getting a molecule and the fingerprint
-                mol, values = conf_gen.generate_conformers(mol)
-                fps = fprints_from_mol(mol, fprint_params=fprint_params)
+                try:
+                    # getting a molecule and the fingerprint
+                    mol, values = conf_gen.generate_conformers(mol)
+                    fps = fprints_from_mol(mol, fprint_params=fprint_params)
 
-                # chose the fingerprint with the lowest energy
-                energies = values[2]
-                fp = fps[np.argmin(energies)].fold(1024)
+                    # chose the fingerprint with the lowest energy
+                    energies = values[2]
+                    fp = fps[np.argmin(energies)].fold(1024)
```

Comment:
Do not use constant 1024, this is a constructor parameter

Reply:
IMO it's better to leave like this. Either 1. we'd need to define a hyperparameter for folding bits - that means we should pass it as an argument to all fingerprint constructors (that requires it) and that is messing with the current helper function for calculating time for our library, or 2. we'd need to use the parameter's value from E3FP instance, which is not directly accessible from the function above.

Reply:
The target dimensionality is the number of bits after folding. I mean, if the default value for E3FP dimensionality is 1024, then it should be 1024 after folding. But for now let's leave this as-is, we can change it later if needed.